common/src/commonMain/kotlin/io/xorum/codeforceswatcher/features/users/redux/reducers/UsersReducer.kt

```diff
@@ -57,6 +57,12 @@ fun usersReducer(action: Action, state: AppState): UsersState {
         is AuthRequests.FetchUserAccount.Success -> {
             newState = newState.copy(userAccount = action.userAccount)
         }
+        is UsersRequests.Destroy -> {
+            newState = newState.copy(
+                    userAccount = null,
```

Comment:
I suppose it's better to make userAccount = null on logOut request

Reply:
Not really.